optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/test/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfigurationTest.java

```diff
@@ -339,7 +336,7 @@ public void constraintVerifierOnDrl() {
                     assertThatCode(() -> {
                         context.getBean(ConstraintVerifier.class).verifyThat();
                     })
-                            .hasMessage("ConstraintVerifier is only supported for ConstraintProviders");
+                            .hasMessage("Cannot provision a ConstraintVerifier because there is ConstraintProvider class.");
```

Comment:
Does this make sense? Shouldn't it be `because there is no`?

Reply:
Agreed; it should be "because there is no ConstraintProvider class"